src/textual/widget.py

```diff
@@ -479,9 +480,11 @@ def render_styled(self) -> RenderableType:
 
         renderable_text_style = parent_text_style + text_style
         if renderable_text_style:
-            renderable = Styled(renderable, renderable_text_style)
+            color = text_style.color
+            bgcolor = text_style.bgcolor
+            renderable = Styled(renderable, Style(color=color, bgcolor=bgcolor))
```

Comment:
There is a shortcut for creating a style from just colors which avoid some rendundant work.

```python
Style.from_colors(text_style.color, text_style.bgcolor)
```

Reply:
TIL, thanks